src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,89 @@
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
```

Comment:
Similarly to the previous comment, the `href` attribute for the previous link is hardcoded to `#prev`. If you are preventing the default action and using onClick to handle the page change, you might not need the `href` attribute at all, or you could use `href='#!'` to indicate that it's a placeholder for a dynamic link.